rj_msgs/CMakeLists.txt

```
@@ -61,6 +61,7 @@ rosidl_generate_interfaces(
   msg/TeamInfo.msg
   msg/Trajectory.msg
   msg/WorldState.msg
+  msg/Position.msg
```

```suggestion
  msg/PositionAssignments.msg
```
this is a better name, less conflict with the "Position" superclass (see position.hpp)